.gitignore

```diff
@@ -1,24 +1,117 @@
-##########################
-### OS RELATED IGNORES ###
-##########################
+# Created by https://www.gitignore.io/api/node,linux,macos,windows
```

Comment:
Missen we niet nog wat IDE stuff?

Reply:
Ik heb Visual Studio Code toegevoegd. Zou wel niet alle IDE's toevoegen want dan gaan we lang bezig zijn. Kunnen er gerust nog toevoegen als er mensen beginnen switchen.